### PR TITLE
Wire up the api server to the privop command, and get port range from environment.

### PR DESCRIPTION
**Pull request by zenhack**
Built atop #12; merge that first. Only the last two commits are part of this pr.

You can actually create vpns via the API now!

I've discovered an issue which I'll open a bug for in a moment.

### CODE REVIEW

File: cmd/hil-vpnd/privops.go [hunk 1]
```diff
@@ -16,9 +22,40 @@ type PrivOps interface {
...
+}
+
+func (PrivOpsCmd) ListVPNs() ([]string, error) {
+	out, err := exec.Command("sudo", "hil-vpn-privop", "list").Output()
+	if err != nil {
+		return nil, err
+	}
+	lines := strings.Split(string(out), "\n")
+
+	// Chop off the trailing empty line, if any:
```
Comment (ianballou) [inline cmd/hil-vpnd/privops.go:55]: Would there ever be a case where there are multiple empty lines? This bit could potentially keep chopping lines until no more are empty

**Review by ianballou — COMMENTED** [review]
Looking good, just a question

**Comment by ianballou** [comment]
Oh also, I don't see the commit here to get the port range from the environment, I'm only seeing it in #15

File: cmd/hil-vpnd/privops.go (hunk 1 above)
Comment (zenhack) [reply cmd/hil-vpnd/privops.go:55]: There shouldn't be; privop just calls fmt.Println in a loop.

**Comment by zenhack** [comment]
@ianballou, looks like I forgot to push -- it's there now.

**Review by naved001 — APPROVED** [review]
looks good to me.

**Comment by zenhack** [comment]
Rebased. @ianballou, just waiting on your signoff now.

**Review by ianballou — APPROVED** [review]
Signing off!

**Pull request merged**